store/sqlstore/compliance_store.go

```diff
@@ -342,3 +342,55 @@ func (s SqlComplianceStore) MessageExport(cursor model.MessageExportCursor, limi
 	}
 	return cposts, cursor, nil
 }
+
+func (s SqlComplianceStore) BlocksExport(cursor model.BlockExportCursor, limit int) ([]*model.BlockExport, model.BlockExportCursor, error) {
+	var version int
+	schemaVersionRow := s.GetReplicaX().QueryRowX("select version from focalboard_schema_migrations")
+	if schemaVersionRow.Err() != nil {
+		return nil, cursor, schemaVersionRow.Err()
+	}
+	schemaVersionRow.Scan(&version)
+	if version != 16 {
```

Comment:
Should this be `version < 16`? The export will continue to work in schema versions above 16 as well, right?

Reply:
I'm wondering the same thing.

Reply:
The idea is to make it work only for the right version, in this case, is the last existing version, I'm not really happy with this part, I have to rethink this.